paywall/src/components/checkout/CheckoutConfirmingModal.tsx

```diff
@@ -0,0 +1,173 @@
+import styled from 'styled-components'
+import React from 'react'
+import { WordMarkLogo } from '../interface/Logo'
+
+import { PaywallConfig, Lock, Account } from '../../unlockTypes'
+import Media from '../../theme/media'
+import CheckoutLock from './CheckoutLock'
+import ActionButton from '../interface/buttons/ActionButton'
+import { LockHeader, LockWrapper } from '../lock/LockStyles'
+
+interface Props {
+  config: PaywallConfig
+  confirmingLock: Lock
+  account: Account | null
+  hideCheckout: (...args: any[]) => any
+}
+
+export const CheckoutConfirmingModal = ({
+  config,
+  confirmingLock,
+  account,
+  hideCheckout,
+}: Props) => {
+  const noTitleLock = {
+    ...confirmingLock,
+  }
+  delete noTitleLock.name
+  return (
+    <React.Fragment>
+      <Header>
+        <Title>
+          {config.icon && <Logo src={config.icon} />}
+          <UnlockedText>Unlocked</UnlockedText>
+        </Title>
+        <h2>Thanks for your purchase!</h2>
+      </Header>
+      <Content>
+        <CheckoutLock
+          lock={noTitleLock}
+          account={account}
+          disabled
+          purchase={() => {}}
+          hideCheckout={hideCheckout}
+        />
+        <div>
+          <p>
+            Your transaction was sent and is currently being confirmed on the
+            Ethereum blockchain. Feel free to start browsing our site ads free!
```

Comment:
```suggestion
            Ethereum blockchain. Feel free to start browsing our site ad-free!
```

Wording nitpick, but I see "ad-free" in common use as opposed to "ads free".

Reply:
This message is specific to Forbes so it should be part of the `unlockProcolConfig` but we should provide a more "gemeric" default (one which does not mention ad free...)